src/core/plugins/swagger-js/index.js

```diff
@@ -22,6 +24,16 @@ module.exports = function({ configs, getConfigs }) {
       },
       serializeRes: Swagger.serializeRes,
       opId: Swagger.helpers.opId
-    }
+    },
+    statePlugins: {
+      configs: {
+        wrapActions: {
+          loaded: (ori, system) => (...args) => {
+            ori(...args)
+            fetch.withCredentials = !!system.getConfigs().withCredentials
```

Comment:
This is a clever way to trigger setting `fetch.withCredentials`, but I think using [`afterLoad`](https://github.com/swagger-api/swagger-ui/blob/master/docs/customization/plugin-api.md#afterload) would be a more robust approach 😄